cluster/environments/azure-simple/main.tf

```diff
@@ -1,61 +1,29 @@
-provider "azurerm" {
-  subscription_id = "${var.subscription_id}"
-  client_id       = "${var.service_principal_id}"
-  client_secret   = "${var.service_principal_secret}"
-  tenant_id       = "${var.tenant_id}"
+terraform {
+  backend "azurerm" {}
 }
 
-# terraform {
-#    backend "azurerm" {
-#    }
-# }
-
-data "azurerm_subscription" "current" {
-  subscription_id = "${var.subscription_id}"
-}
-
-module "vnet" {
-  source = "../../azure/vnet"
-
-  resource_group_name     = "${var.resource_group_name}"
-  resource_group_location = "${var.resource_group_location}"
-  subnet_names            = ["${var.cluster_name}-aks-subnet"]
-
-  tags = {
-    environment = "azure-simple"
-  }
+resource "azurerm_resource_group" "cluster_rg" {
+  name     = "${var.resource_group_name}"
+  location = "${var.resource_group_location}"
 }
 
-module "aks" {
-  source = "../../azure/aks"
+module "aks-gitops" {
+  source = "github.com/timfpark/bedrock//cluster/azure/aks-gitops?ref=aks_gitops"
```

Comment:
Needs link for url when merged to master

Reply:
Yes - agreed - will fix this on merge to Microsoft/bedrock